<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherSelftestDialog</class>
 <widget class="QDialog" name="OtherSelftestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OtherSelftestDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWbcBurn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>WBC Burn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRbcBurn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>RBC Burn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConst">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Const</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbHgbLed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>HGB Led</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBuzzer">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Buzzer</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
